sdk/storage/Azure.Storage.DataMovement/src/Shared/TransferManagerFactory.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Azure.Storage.DataMovement
+{
+    internal class TransferManagerFactory
```

Comment:
Is this only for tests? Maybe we should move it into test folder/assembly?

Reply:
Done